Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/TextMetadataTableRow.java

```diff
@@ -101,6 +108,32 @@ public void setValue(String value) {
         this.value = value;
     }
 
+    /**
+     * Get date.
+     *
+     * @return value of date
+     */
+    public Date getDate() {
+        if (Objects.isNull(date) && Objects.nonNull(getValue())) {
+            try {
+                date = new SimpleDateFormat("yyyy-mm-dd").parse(getValue());
+            } catch (ParseException e) {
+                logger.error(e.getLocalizedMessage());
```

Comment:
For error logger stack trace is already quiet wanted:

`logger.error(e.getLocalizedMessage(), e);`